packages/lesswrong/components/votes/lwReactions/NamesAttachedReactionsVoteOnComment.tsx

```diff
@@ -32,6 +32,9 @@ const styles = (theme: ThemeType) => ({
     alignItems: "center",
     whiteSpace: "nowrap",
   },
+  agreementButtons: {
+    marginLeft: 8
```

Comment:
affects things outside the feed?

Reply:
This is moved from elsewhere, preserving things effectively here but letting me have them not apply elsewhere